storage/innobase/handler/ha_innodb.cc

```diff
@@ -19977,9 +19979,11 @@ ib_logf(
 		break;
 	case IB_LOG_LEVEL_ERROR:
 		sql_print_error("InnoDB: %s", str);
+		sd_notifyf(0, "STATUS=InnoDB: Error: %s", str);
 		break;
```

Comment:
Alright, lets give it a try. After all it isn't hot path and I hope normally systemd won't be flooded by insane amount of such errors.